wagmi.config.ts

```diff
@@ -139,6 +139,27 @@ const contracts: ContractConfig[] = [
       [nitroTestnodeL2.id]: '0x3BaF9f08bAD68869eEdEa90F2Cc546Bd80F1A651',
     },
   },
+  {
+    name: 'RollupCreator',
+    version: '2.1',
+    address: {
+      // mainnet L1
+      [mainnet.id]: '0x8c88430658a03497D13cDff7684D37b15aA2F3e1',
+      // mainnet L2
+      [arbitrumOne.id]: '0x79607f00e61E6d7C0E6330bd7E9c4AC320D50FC9',
+      [arbitrumNova.id]: '0x9B523BF5F77e8d90e0E9eb0924aEA6E40B081aE6',
+      [base.id]: '0x091b8FC0F48613b191f81009797ce55Cf97Af7C8',
+      // testnet L1
+      [sepolia.id]: '0xfb774ea8a92ae528a596c8d90cbcf1bdbc4cee79',
+      [holesky.id]: '0x03c70F125Df471E4fd0515ca38504edFE6900F19',
+      // testnet L2
+      [arbitrumSepolia.id]: '0xd2Ec8376B1dF436fAb18120E416d3F2BeC61275b',
+      [baseSepolia.id]: '0x6e244cD02BBB8a6dbd7F626f05B2ef82151Ab502',
+      // local nitro-testnode (on "release" branch with --tokenbridge --l3node --l3-token-bridge flags)
+      [nitroTestnodeL1.id]: '0x82a3c114b40ecf1fc34745400a1b9b9115c33d31',
+      [nitroTestnodeL2.id]: '0x4287839696d650a0cf93b98351e85199102335d0',
+    },
+  },
```

Comment:
this is also hardcoded as `rollupCreatorAddress` in `RollupCreator/v2.1.ts`, possible to consolidate?